examples/gke-a3-highgpu.yaml

```diff
@@ -65,4 +65,12 @@ deployment_groups:
       machine_type: a3-highgpu-8g
       autoscaling_total_min_nodes: 2
       zones: [$(vars.zone)]
-    outputs: [instructions]
```

Comment:
please revert this deletion

curious, are you branching from main instead of develop?